Dockerfile

```
@@ -0,0 +1,6 @@
+FROM openjdk:21-jdk-slim
+LABEL authors="DainXOR"
+
+EXPOSE 8080
+ADD target/banco_udea.jar banco_udea-0.0.1-SNAPSHOT.jar
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)